indigo/distribution.yaml

```diff
@@ -8399,6 +8399,23 @@ repositories:
       type: git
       url: https://github.com/silviomaeta/plot_util.git
       version: master
+  plotjuggler-release:
+    doc:
+      type: git
+      url: https://github.com/facontidavide/PlotJuggler.git
+      version: 0.8.0
```

Comment:
The version needs to be a valid branch name, likely `master`.

Same for the `source` entry below.